packages/electron-builder/templates/nsis/boringInstaller.nsh

```diff
@@ -8,9 +8,9 @@
 !ifndef BUILD_UNINSTALLER
   Function StartApp
     ${if} ${Updated}
-      ${StdUtils.ExecShellAsUser} $0 "$startMenuLink" "open" "--updated"
+      ${StdUtils.ExecShellAsUser} $0 "$INSTDIR\${APP_EXECUTABLE_FILENAME}" "open" "--updated"
     ${else}
-      ${StdUtils.ExecShellAsUser} $0 "$startMenuLink" "open" ""
+      ${StdUtils.ExecShellAsUser} $0 "$INSTDIR\${APP_EXECUTABLE_FILENAME}" "open" ""
```

Comment:
@MariaDima We cannot do it. We must open app usng start menu link because only in this case Application User Model ID (https://msdn.microsoft.com/en-us/library/windows/desktop/dd378459(v=vs.85).aspx) will be applied correctly. We set this *** MS id only for link.

Not clear for me — can we set it for executable or not. To investigate.